### PR TITLE
[[ IDE Testing ]] Make sure the test runner is only initiated by its own openStack message

### PR DESCRIPTION
**Pull request by livecodeali**
...and its own startup message.

### CODE REVIEW

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay 14d93ca

**Comment by livecode-vulcan** [comment]
💙 review by @livecodefraser ok 14d93ca391d32297ac7c986af2dd532724a48566

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay 0ce38f1

**Comment by livecode-vulcan** [comment]
💙 review by @livecodefraser ok 0ce38f129f5258d22a7ea71490cf057a09da8d71

File: tests/_testrunner.livecodescript [hunk 1]
```diff
@@ -20,15 +20,26 @@ along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
...
+	   else
+		  send "TestRunnerMain" to me in 0
+	   end if
    end if
 end startup
 
 on openStack
-   send "TestRunnerInvokeAll" to me in 0
+   if the long id of the target is the long id of this card of me then
+	   send "TestRunnerInvokeAll" to me in 0
```
Comment (montegoulding) [inline tests/_testrunner.livecodescript:41]: @livecodeali how would you feel about me making a little UI for the test runner. I think I could come up with some features that could save some time in test writing

Comment (livecodeali) [inline tests/_testrunner.livecodescript:41]: @montegoulding That sounds like a good idea! Especially if it helps the community to write and add tests.

Comment (peter-b) [inline tests/_testrunner.livecodescript:41]: @montegoulding Oooh, with graphical visualisation of the results? 👯

**Pull request merged**